templates/webapi/test/overview.html.ep

```diff
@@ -124,6 +124,12 @@
                             <input type="text" class="form-control" name="test" placeholder="any" id="filter-test">
                         </div>
                     </div>
+                    <div class="col-5">
+                        <div class="form-group">
+                            <strong>Test module</strong> (regular expression)
+                            <input type="text" class="form-control" name="module_re" placeholder="any" id="filter-module-re">
+                        </div>
+                    </div>
```

Comment:
A Bootstrap column has 12 rows, you're using 15 here if i'm not mistaken.

Reply:
thx, learned something new :D